frontend/src/components/Navbar.tsx

```diff
@@ -20,6 +21,13 @@ export default function Navbar({ _currentPage }: NavbarProps): JSX.Element {
     setIsOpen(!isOpen);
   };
 
+  const handleLogout = () => {
+    Cookies.remove("auth-storage");
+    Cookies.remove("refreshToken");
+    
```

Comment:
Here, it would also be good to have:
`  const logout = useAuthStore((state) => state.logout);`

to remove the tokens from the global state "Zustand" Manager.